test/src/problems/FailingProblem.C

```diff
@@ -17,21 +17,24 @@ InputParameters
 FailingProblem::validParams()
 {
   InputParameters params = FEProblem::validParams();
-  params.addRequiredParam<unsigned int>("fail_step", "The timestep to fail");
+  params.addRequiredParam<std::vector<int>>("fail_steps", "The timestep(s) to fail");
   return params;
 }
 
 FailingProblem::FailingProblem(const InputParameters & params)
-  : FEProblem(params), _failed(false), _fail_step(getParam<unsigned int>("fail_step"))
+  : FEProblem(params), _fail_steps(getParam<std::vector<int>>("fail_steps"))
```

Comment:
```suggestion
  : FEProblem(params), _fail_steps(getParam<std::vector<unsigned int>>("fail_steps"))
```

Reply:
This was actually intentional, since t_step is a regular int for some reason, compiler doesn't like you doing compares with unsigned vs signed int. better to avoid the compiler warning i think